docs/sessions.md

```diff
@@ -56,12 +57,23 @@ Server Keys
 
 The contents of the cookie are encrypted before the cookie is sent. Suave's default configuration generates a new server key each time the server is restarted. While this is not _wrong_, users would likely get quite annoyed if they lost their state because the server was restarted. Additionally, specifying a server key lets load-balanced servers access the same information.
 
-Continuing our example from above... _(This should not be used on production servers; a key should be generated and provided via a configuration file.)_
+That being said, specifying a good server is important for security as well. You can use [a web-based service](https://www.random.org/strings/) to generate 32 characters of random text. You can also run the following, either in an .fsx file (for the full .NET Framework) or in a console app (for .NET Core), to generate a UTF-8 key. (This requires no Suave libraries.)
+
+{% highlight fsharp %} 
+let rnd = System.Random ()
```

Comment:
This random generator is insecure.

Why can't we call the built in `generateKey` function? https://github.com/SuaveIO/suave/blob/master/src/Suave/Utils/Crypto.fs#L61